YTSpammerPurge.py

```diff
@@ -1380,7 +1380,7 @@ def scan_video(miscData, config, filtersDict, scanVideoID, videosToScan=None, cu
 
     # User wants to automatically delete with no user intervention
     elif config['delete_without_reviewing'] == True:
-      if userNotChannelOwner == True:
+      if userNotChannelOwner == True and moderator_mode == False:
```

Comment:
Hold on here... why are boolean comparisons being done here?

```suggestion
    elif config['delete_without_reviewing']:
      if userNotChannelOwner and not moderator_mode:
```